src/nikhilMJun20/PalindromeString.java

```diff
@@ -0,0 +1,42 @@
+/*Verify given String is palindrome or not. 
+Input: radar
+Output : Given string radar is palindrome.
+
+Input: technocredits
+Output: Given string technocredits is not palindrome.
+*/
+
+package nikhilMJun20;
+
+import java.util.Scanner;
+
+public class PalindromeString {
+	void checkPalindromeString(String str) {
+		boolean flag = true;
+		str = str.toLowerCase();
+		for(int leftIndex=0, rightIndex=str.length()-1; leftIndex<=rightIndex; leftIndex++,rightIndex--) {
```

Comment:
-- Your code and logic looks good to me.
--However I think you can simplify it a little.
--Refer below code:
String reverse="", originalName = name;
		for(int index=name.length()-1;index>=0;index--) {
			reverse = reverse+name.charAt(index);		
		}
		if(originalName.equals(reverse))
			System.out.println(" Given String " +originalName +" is Palindrome");
		else
			System.out.println(" Given String " +originalName +" is not Palindrome");

Reply:
->Yes, it is a simple and string reverse logic. But, in my logic I do not required an additional string for keeping the reversed string and at the first mismatch - where one incrementing and one decrementing index pointer is used, I will be able to tell that the string is not a palindrome.
->Thanks for the review comment.

Reply:
@VaishnaviVJun20 Nikhil's logic is correct and as expected. Not required another string to store all characters in reverse order.